assets/index.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.jpg'{
+    const value: any;
```

Comment:
stick?

Reply:
Dit is om bepaalde filetypes te kunnen gebruiken

Reply:
maar kan het dan geen `:string` zijn oid?

Reply:
https://stackoverflow.com/a/51163365